api/tests/unittest/test_feeds.py

```diff
@@ -150,7 +150,7 @@ def test_gtfs_feeds_get(client: TestClient, mocker):
 
     feed_mdb_10 = Database().get_query_model(Gtfsfeed).filter(Gtfsfeed.stable_id == "mdb-10").first()
     assert response.status_code == 200, f"Response status code was {response.status_code} instead of 200"
-    response_gtfs_feed = response.json()[0]
+    response_gtfs_feed = response.json()[3]
```

Comment:
I believe if you add the feed data to test/integration/test_data/extra_test_data.json you will not have to change this test.

Reply:
done ✅